README.md

```diff
@@ -2,7 +2,215 @@
+| 2.0-node12                | [Dockerfile]()  | Debian 9     |
+| 2.0-node12-slim           | [Dockerfile]()  | Debian 9     |
+| 2.0-node12-appservice, 2.0-node12-appservice-quickstart | [Dockerfile]() | Debian 9  |
+
+Linux alpine Tags
+
+| Tags                         | Dockerfile     | OS Version    |
+| ---------------------------- | -------------- | ------------- |
+| 2.0-alpine, 2.0-node8-alpine | [Dockerfile]() | Alpine 3.8    |
+| 2.0-node10-alpine            | [Dockerfile]() | Alpine 3.8    |
+| 2.0-node12-alpine            | [Dockerfile]() | Alpine 3.8    |
+
+#### Powershell
+
+`mcr.microsoft.com/azure-functions/powershell`
+
+Linux amd64 Tags
+
+| Tags                           | Dockerfile      | OS Version   |
+| ------------------------------ | --------------- | ------------ |
+| 2.0, 2.0-powershell6           | [Dockerfile]()  | Debian 9     |
+| 2.0-slim, 2.0-powershell6-slim | [Dockerfile]()  | Debian 9     |
+| 2.0-appservice, 2.0-appservice-quickstart, 2.0-powershell6-appservice, 2.0-powershell6-appservice-quickstart | [Dockerfile]() | Debian 9  |
+
+Linux alpine Tags
+
+| Tags                               | Dockerfile     | OS Version    |
+| ---------------------------------- | -------------- | ------------- |
+| 2.0-alpine, 2.0-powershell6-alpine | [Dockerfile]() | Alpine 3.8    |
+
+
+#### Java
+
+`mcr.microsoft.com/azure-functions/java`
+
+Linux amd64 Tags
+
+| Tags                     | Dockerfile      | OS Version   |
+| ------------------------ | --------------- | ------------ |
+| 2.0, 2.0-java8           | [Dockerfile]()  | Debian 9     |
+| 2.0-slim, 2.0-java8-slim | [Dockerfile]()  | Debian 9     |
+| 2.0-appservice, 2.0-appservice-quickstart, 2.0-java8-appservice, 2.0-java8-appservice-quickstart | [Dockerfile]() | Debian 10 |
+| 2.0-java8-build          | [Dockerfile]()  | Debian 9     |
+
+Linux alpine Tags
+
+| Tags                         | Dockerfile     | OS Version    |
+| ---------------------------- | -------------- | ------------- |
+| 2.0-alpine, 2.0-java8-alpine | [Dockerfile]() | Alpine 3.8    |
+
+
+### Python
+
+`mcr.microsoft.com/azure-functions/python`
+
+Linux amd64 Tags
+
+| Tags                      | Dockerfile      | OS Version   |
+| ------------------------- | --------------- | ------------ |
+| 2.0, 2.0-python3.6           | [Dockerfile]()  | Debian 9     |
+| 2.0-slim, 2.0-python3.6-slim | [Dockerfile]()  | Debian 9     |
+| 2.0-appservice, 2.0-appservice-quickstart, 2.0-python3.6-appservice, 2.0-python3.6-appservice-quickstart | [Dockerfile]() | Debian 9  |
+| 2.0-python3.6-buildenv       | [Dockerfile]()  | Debian 9     |
+| 2.0-python3.7                | [Dockerfile]()  | Debian 9     |
+| 2.0-python3.7-slim           | [Dockerfile]()  | Debian 9     |
+| 2.0-python3.7-appservice, 2.0-python3.7-appservice-quickstart | [Dockerfile]() | Debian 9  |
+| 2.0-python3.7-buildenv       | [Dockerfile]()  | Debian 9     |
+| 2.0-python3.8                | [Dockerfile]()  | Debian 9     |
```

Comment:
we don't have python 3.8 support in v2

Reply:
good catch. I copied the section from the v3 above and updated the versions.